components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -736,6 +780,8 @@ func (backend *ES2Backend) GetControlListItems(ctx context.Context, filters map[
 		return nil, err
 	}
 
+	LogQueryPartMin(esIndex, searchResult.Aggregations, fmt.Sprintf("%s searchResult aggs", myName))
```

Comment:
Should this logging be removed?

Reply:
No, this is a pattern we use in a bunch of places in the elasticsearch code.  First, we log the query itself and then we log the actual result of the query.  It doesn't litter the logs because it's set to debug level logging.